client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -203,10 +206,22 @@ class PurchasesListing extends Component {
 		if ( purchase.autoRenew && ! shouldAddPaymentSourceInsteadOfRenewingNow( purchase ) ) {
 			label = translate( 'Renew now' );
 		}
+		const isProductOwner = Boolean( purchase && purchase.userId === currentUserId );
```

Comment:
Noting that above ([L179](https://github.com/Automattic/wp-calypso/pull/59549/files#diff-c279901f1042db1f48e3b67424fa878e5f7f3eb4f12ac4a6f70133dbd7ab471cR179)) we already check for the presence of `purchase`, so we could remove the `purchase &&` check here.

Reply:
Done here https://github.com/Automattic/wp-calypso/pull/59749/files#r778597421

Thanks

Reply:
Thank you 🙌